app/assets/javascripts/covid19/src/webapp/components/structures-table/StructuresTable.tsx

```diff
@@ -31,6 +31,12 @@ export const StructuresTable: React.FC<StructuresTableProps> = React.memo(props
 
     const [isDialogOpen, { enable: openDialog, disable: closeDialog }] = useBooleanState(false);
     const [detailsOptions, setDetailsOptions] = React.useState<FieldStructure>();
+    const [sortModel, setSortModel] = React.useState<GridSortModel>([
+        {
+            field: "emdb",
+            sort: "desc" as GridSortDirection,
```

Comment:
[ultra-minor] No need for `as GridSortDirection` because of the `FieldStructure` hint at useState. No need to change.